### PR TITLE
Improved: code by adding loader when logging out(#60)

### PR DESCRIPTION
**Pull request by Ritika-Patel08**
### Related Issues

Related Issue #60

### Short Description and Why It's Useful

Added loader when logging out such that users will have a visual cue that their action is in progress which will definitely enhance the user experience. 

### Screenshots of Visual Changes before/after (If There Are Any)

**IMPORTANT NOTICE** - Remember to add changelog entry

### Contribution and Currently Important Rules Acceptance

- [x] I read and followed [contribution rules](https://github.com/hotwax/job-manager#contribution-guideline)

### CODE REVIEW

File: src/App.vue [hunk 1]
```diff
@@ -87,6 +90,22 @@ export default defineComponent({
...
+  async mounted() {
+    this.loader = await loadingController
+      .create({
+        message: this.$t("Click the backdrop to dismiss."),
+        translucent: true,
+        backdropDismiss: true
+      });
+    emitter.on('presentLoader', this.presentLoader);
+    emitter.on('dismissLoader', this.dismissLoader);
+
```
Comment (amansinghbais) [inline src/App.vue:102]: Removed this unnecessary space.

File: src/store/auth.ts [hunk 2]
```diff
@@ -87,6 +88,7 @@ export const useAuthStore = defineStore('authStore', {
       }
     },
     async logout(payload?: any) {
+      emitter.emit("presentLoader",{ message: "Logging out...", backdropDismiss: false });
```
Comment (amansinghbais) [inline src/store/auth.ts:91]: Please check in the translation for this message entry.

File: src/views/Home.vue [hunk 3]
```diff
@@ -86,6 +86,7 @@ import {
 import { useAuthStore } from '@/store/auth';
 import { useRouter } from "vue-router";
 import { goToOms } from '@hotwax/dxp-components'
+import emitter from "@/event-bus";
```
Comment (amansinghbais) [inline src/views/Home.vue:89]: Remove import for this unused emitter

Comment (Ritika-Patel08) [reply src/views/Home.vue]: done.

File: src/App.vue (hunk 1 above)
Comment (Ritika-Patel08) [reply src/App.vue]: done.

**Review by amansinghbais — APPROVED** [review]

**Review by amansinghbais — APPROVED** [review]

File: src/store/auth.ts (hunk 2 above)
Comment (ymaheshwari1) [inline src/store/auth.ts:91]: Move this inside the if condition of unauthorised check

**Review by ymaheshwari1 — CHANGES_REQUESTED** [review]

**Review by ymaheshwari1 — APPROVED** [review]

**Pull request merged**